src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -176,7 +176,7 @@ export const JiraAtlassianConnectGet = async (_: Request, res: Response): Promis
 			"signed-install": true
 		},
 		name: APP_NAME,
-		description: "Connect your code and your project with ease.",
+		description: "Connect your code and your projects with ease.",
```

Comment:
This is trying to trigger an update for ALL the jira sites. And according to the doc, it will [rotate a new shared secrets](https://developer.atlassian.com/cloud/jira/platform/connect-app-descriptor/#lifecycle).

I am hoping trigger an update for ALL jira site may help recover existing client key for the sites that's 404/unavailable/etc

But I am not sure. Need some advices from you.

ps: Although I don't think there should be any harm though.

Reply:
Yeah, not sure either, but given we did that multiple times already that should be fine I guess. But would be nice to know for sure, maybe ping ecosystem, or just add some debug logging? To know for sure for the future...

Reply:
Pretty sure that connect automatically go through customers every few weeks.  you could ask the connect team about this, but remember when we were doing the DB enryption updates with cryptor?  Like half our customers had already moved after a couple weeks in prod.

I'm not sure you need to update the connect json for this update to trigger.

Reply:
Yup, agree. Already ask the connect team and will update this.

Reply:
Confirm better make a change to trigger it.

Reply:
Get feedback from connect team. They can manually re-trigger the events for some/all sites for our app. So reverted the changes here.